api/deployments/models/component_deployment.go

```diff
@@ -268,6 +269,29 @@ type ComponentSummary struct {
 	SkipDeployment bool `json:"skipDeployment,omitempty"`
 }
 
+// ReplicaType The replica type
+type ReplicaType int
+
+const (
+	// JobManager Replica of a Radix job-component scheduler
+	JobManager ReplicaType = iota
```

Comment:
Can we change the type to string and hardcode the string values?

Reply:
Yes, we do this in many cases in radix-operator. This way it looks more controllable (enum-like) and follows other cases in api, so I suggest we keep it as this.
<img width="813" alt="image" src="https://github.com/equinor/radix-api/assets/702860/e0eae27a-4284-4fd3-b8e7-3ad5a9273899">